docs/ref/contrib/admin/index.txt

```diff
@@ -1077,6 +1077,15 @@ subclass::
     ``prepopulated_fields`` doesn't accept ``DateTimeField``, ``ForeignKey``,
     ``OneToOneField``, and ``ManyToManyField`` fields.
 
+    ``prepopulated_fields`` removes stop words::
```

Comment:
I don't think the complete list needs to be mentioned. I would modify a couple paragraphs above to say, "(e.g. substituting dashes for spaces and  removing various English stop words such as 'a', 'an', 'as', and similar).

Reply:
Ok, I will add in these changes.